pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -42,31 +37,22 @@ func (h *Helper) InstallFlags(flags *pflag.FlagSet) {
 // from the local dockercfg file
 func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfiguration, bool) {
 	glog.V(3).Infof("Locating docker auth for image %s and type %s", imageName, authType)
-	var dockercfgPath string
-	if pathForAuthType := os.Getenv(authType); len(pathForAuthType) > 0 {
-		dockercfgPath = GetDockercfgFile(pathForAuthType)
```

Comment:
@bparees agreed, I have rewrote the code, as `GetDockerAuth` returns completely file path, so I extract the directory and the file, and then call public function. Is that reasonable?